docs/source/user_guide/installation/install-source-windows.rst

```diff
@@ -0,0 +1,382 @@
+
+Manual install
+~~~~~~~~~~~~~~
+
+We recommend installing PyBaMM within a virtual environment to avoid altering any distribution of Python files.
+
+In the same Command Prompt shell where you've set the environment variables, install ``virtualenv`` by executing the following command:
+
+.. code:: bash
+
+   python -m pip install virtualenv
+
+Create a virtual environment ``venv`` within the PyBaMM directory:
+
+.. code:: bash
+
+   python -m virtualenv venv
+
+You can then “activate” the environment using:
+
+.. code:: text
+
+   venv\Scripts\activate.bat
+
+Now, all the calls to pip described below will install PyBaMM and its
+dependencies into the environment ``venv``. When you are ready to exit
+the environment and go back to your original system, just type:
+
+.. code:: bash
+
+   deactivate
+
+
+From the ``PyBaMM/`` directory inside the virtual environment, you can install PyBaMM using
+
+.. code:: bash
+
+	  pip install .
+
+If you want to install PyBaMM in an "editable mode", use the following command:
+
+.. code:: bash
+
+	  pip install -e .
+
+If you intend to contribute to the development of PyBaMM, it is convenient to
+install in "editable mode", along with all the optional dependencies and useful
+tools for development and documentation.
+
+Due to the ``--no-build-isolation`` flag in the "editable mode" command, you first need to install the build-time dependencies inside the virtual environment:
+
+.. code:: bash
+
+	pip install scikit-build-core==0.10.3 pybind11
+
+You can now install PyBaMM in "editable mode" for development using the following commands:
+
+.. code:: bash
+
+	  pip install --no-build-isolation --config-settings=editable.rebuild=true -ve .[all,dev,docs]
+
+Before you start contributing to PyBaMM, please read the `contributing
+guidelines <https://github.com/pybamm-team/PyBaMM/blob/develop/CONTRIBUTING.md>`__.
+
+Running the tests
+-----------------
+
+Using Nox (recommended)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use ``Nox`` to run the unit tests and example notebooks in isolated virtual environments.
+
+Make sure to install ``graphviz`` using the `Chocolatey <https://chocolatey.org/>`_ package manager or by following the instructions on the `graphviz website <https://graphviz.org/download/>`_ to pass all the tests. Otherwise, you can safely ignore the failed tests needing ``graphviz``.
+
+The default command
+
+.. code:: bash
+
+	nox
+
+will run pre-commit and run the unit tests.
+This can take several minutes.
+
+To just run the unit tests, use
+
+.. code:: bash
+
+	nox -s unit
+
+Similarly, to run the integration tests, use
+
+.. code:: bash
+
+	nox -s integration
+
+Finally, to run the unit and the integration suites sequentially, use
+
+.. code:: bash
+
+	nox -s tests
+
+Using the test runner
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can run unit tests for PyBaMM inside the virtual environment using
+
+.. code:: bash
+
+	  # in the PyBaMM/ directory
+	  python run-tests.py --unit
```

Comment:
We removed the `run-tests.py` file in favour of `pytest` commands, could you change this? You may refer to them from the Contributing guide (from the `develop` branch).